dbux-data/src/dataProviderUtil.js

```diff
@@ -269,15 +269,24 @@ export default {
   /** @param {DataProvider} dp */
   doesTraceHaveValue(dp, traceId) {
     const trace = dp.util.getValueTrace(traceId);
-    const { value } = trace;
-    if (value === undefined) {
-      const valueRef = dp.util.getTraceValueRef(traceId);
-      if (!valueRef) { // || valueRef.value === undefined) {
-        // TODO: better distinguish between existing and non-existing values
-        return false;
-      }
+    // valueId might be `0`, cannot simply check `'valueId' in trace`
```

Comment:
Comment is referring to `valueId` instead of `value`.

Also, a suggestion for wording: "`value` might be a legitimate falsy value"